src/lumigo_tracer/auto_tag/auto_tag_event.py

```diff
@@ -48,13 +48,25 @@ def auto_tag(event: dict):
                     add_execution_tag(key, headers[key])
 
 
+class ConfigurationHandler(EventAutoTagHandler):
+    @staticmethod
+    def is_supported(event) -> bool:
+        return bool(event) and any(key in event for key in Configuration.auto_tag)
+
+    @staticmethod
+    def auto_tag(event: dict):
+        for key in Configuration.auto_tag:
+            if key in event:
+                add_execution_tag(key, event[key])
```

Comment:
Notice: I think that `execution_tag` support only strings, here the value might not be a string.

Reply:
currect, but `add_execution_tag` stringify the input. I'll add a test 👌